configs/orchestrator/templates/cloud-secrets.yaml

```diff
@@ -37,3 +37,48 @@ data:
   {{if .Values.clouds.aws.benchmarkOSChannel }}
   BENCHMARK_OS_CHANNEL: {{ .Values.clouds.aws.benchmarkOSChannel | b64enc }}
   {{end}}
+  {{if .Values.clouds.oci.userOCID }}
+  USER_OCID: {{ .Values.clouds.oci.userOCID | b64enc}}
+  {{end}}
+  {{if .Values.clouds.oci.OCIPrivateKey}}
+  PRIVATE_KEY_PATH: {{ "~/.ssh/oci_private.pem" | b64enc }}
+  {{end}}
+  {{if .Values.clouds.oci.fingerprint }}
+  FINGERPRINT: {{ .Values.clouds.oci.fingerprint | b64enc}}
+  {{end}}
+  {{if .Values.clouds.oci.tenancyID }}
+  TENANCY_ID: {{ .Values.clouds.oci.tenancyID | b64enc}}
+  {{end}}
+  {{if .Values.clouds.oci.compartmentID }}
+  COMPARTMENT_ID: {{ .Values.clouds.oci.compartmentID | b64enc}}
+  {{end}}
+  {{if .Values.clouds.oci.adNumber }}
+  AD_NUMBER: {{ .Values.clouds.oci.adNumber | b64enc}}
+  {{end}}
+  {{if .Values.clouds.oci.route53Zone }}
+  AWS_ROUTE53_ZONE: {{ .Values.clouds.oci.route53Zone | b64enc}}
+  {{end}}
+  {{if .Values.clouds.oci.route53ZoneID }}
+  AWS_ROUTE53_ZONE_ID: {{ .Values.clouds.oci.route53ZoneID | b64enc}}
+  {{end}}
+  {{if .Values.clouds.oci.benchmarkWorkerCount }}
+  BENCHMARK_WORKER_COUNT: {{ .Values.clouds.oci.benchmarkWorkerCount | b64enc}}
```

Comment:
Why is this (and the other worker related values) in the secrets part?

Reply:
Moved the non-secret values to a configmap.